examples/2d/plot_filters.py

```diff
@@ -54,29 +54,31 @@ def colorize(z):
     filter_c = np.fft.fftshift(filter_c)
     axs[i // L, i % L].imshow(colorize(filter_c))
     axs[i // L, i % L].axis('off')
-    axs[i // L, i % L].set_title("$j = {}$ \n $\\theta={}".format(i // L, i % L))
+    axs[i // L, i % L].set_title("$j = {}$ \n $\\theta={}$".format(i // L, i % L))
     i = i+1
 
-
-# Add blanks for pretty display
-for z in range(L):
-    axs[i // L, i % L].axis('off')
-    i = i+1
+fig.suptitle("Wavelets for each scales $j$ and angles $\\theta$ used" #, with the corresponding low-pass filter."
+                 "\n The contrast corresponds to the amplitude and the color to the phase.", fontsize=13)
+fig.show()
 
 ###############################################################################
 # Lowpass filter
-# ----------------
-# We finally display the Gaussian filter.
+# --------------
+# We finally display the low-pass filter.
+plt.figure()
+plt.rc('text', usetex=True)
+plt.rc('font', family='serif')
+plt.axis('off')
+plt.set_cmap('gray')
+
+
 f_r = filters_set['phi'][0][...,0].numpy()
 f_i = filters_set['phi'][0][..., 1].numpy()
 f = f_r + 1j*f_i
 filter_c = fft2(f)
 filter_c = np.fft.fftshift(filter_c)
-axs[J, L // 2].imshow(colorize(filter_c))
-
-# Final caption.
-fig.suptitle("Wavelets for each scales $j$ and angles $\\theta$ used, with the corresponding low-pass filter."
-             "\n The contrast corresponds to the amplitude and the color to the phase.", fontsize=13)
-
-
-plt.show()
+plt.suptitle("The corresponding low-pass filter."
```

Comment:
The corresponding low-pass filter, also known as scaling function.
Color saturation and color hue respectively denote complex magnitude and complex phase.